.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,7 +2,7 @@
 name: Feature / Improvement Request
 about: Request an improvement or a new feature.
 title: ''
-labels: 'enhancement'
+labels: ''
 assignees: ''
```

Comment:
We have an `Type: Improvement` bug tag, could use that.

Reply:
There's also one for features, so it seems like it should be up to the issue-opener to select which one they want or I should split it up into two templates?

Reply:
Ah, I see. I think the developers are probably best placed to decide. So default to `enhancement` and one of us can change it if its a bigger change.